tests/testthat/test-bart.R

```diff
@@ -32,4 +32,22 @@ test_that("dbarts + predict() works", {
     predict(x, newdata = head(mtcars))[1],
     predict(res, newdata = head(mtcars))[1]
   )
+  expect_equal(
+    mean(predict(x, newdata = head(mtcars), type = "ppd")),
+    mean(predict(res, newdata = head(mtcars), type = "ppd")),
+    tolerance = 0.1
+  )
+})
+
+test_that("bart() from parsnip + predict() works", {
+  library(parsnip)
+  spec <- bart(mode = "regression", trees = 5)
+  res <- fit(spec, mpg ~ ., mtcars)
+  x <- butcher(res)
+
+  expect_equal(
```

Comment:
I would add some tests with `predict(type = "conf_int")` and `predict(type = "pred_int")` too (just in case).

Reply:
Thanks @topepo!